sb_vision/distance_finding.py

```diff
@@ -0,0 +1,105 @@
+import functools
+from pathlib import Path
+from typing import Tuple, Dict
+
+import cv2
+from lxml import etree
+import re
+import numpy as np
+
+
+def _get_text(element: etree.Element):
```

Comment:
This doesn't really return "text"; is there a better name for it?

Reply:
fixed in 304abe6